### ISSUE TITLE
Correct Syntax/Argument for --LHR ?

### ISSUE THREAD

**Issue by nrpeyton** [Lolliedieb/lolMiner-releases#1482]
Hi,

1.48 is excellent, good work!! Amazing!!

But, I need help with this: (See pictures below).
![lhr_arg](https://user-images.githubusercontent.com/98381240/161495432-2bcca3d0-0dbb-42e8-846c-204e5561ae5b.png)

.

As you can see, I created my own .bat file, but see this picture below: (why it still wasting time)?

.
![checking LHR](https://user-images.githubusercontent.com/98381240/161495683-ab6067c2-d4d0-464e-82e0-b1b05746e11c.png)

**Comment by rcerebro** [comment]
> Hi,
> 
> 1.48 is excellent, good work!! Amazing!!
> 
> But, I need help with this: (See pictures below). ![lhr_arg](https://user-images.githubusercontent.com/98381240/161495432-2bcca3d0-0dbb-42e8-846c-204e5561ae5b.png)
> 
> .
> 
> As you can see, I created my own .bat file, but see this picture below: (why it still wasting time)?
> 
> . ![checking LHR](https://user-images.githubusercontent.com/98381240/161495683-ab6067c2-d4d0-464e-82e0-b1b05746e11c.png)

lhrtune 0 (ZERO)

**Comment by nrpeyton** [comment]
Yeah, I know you can set a specific number, but I want the full 1.48 LHR improvements.

I assumed it worked like this:

**0** = FHR _(LHR off)_
**auto** = LHR card, auto-tune
**77, 78 etc** = LHR card, no auto

?

**Comment by igralec84** [comment]
For me it auto-tunes no matter what you specify, it's like only auto and off work.
